src/api/courses.js

```diff
@@ -211,15 +231,50 @@ router.post(
   ],
   safeAsync(async (req, res, _next) => {
     const { name, shortcode } = matchedData(req)
+    const { isUnlisted } = req.body
```

Comment:
Can you add some validation for these values with `check(...)` above, like we do for `name` and `shortcode`? We mostly want to check that they're booleans. Also, providing defaults in case they aren't specified would be great.